services/testspace/testspace-test-count.tester.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const Joi = require('joi')
+const t = (module.exports = require('../tester').createServiceTester())
+const { isMetric } = require('../test-validators')
+const isMetricAllowZero = Joi.alternatives(
+  isMetric,
+  Joi.number().valid(0).required()
+)
+
+t.create('Total')
+  .get('/total/swellaby/swellaby:testspace-sample/main.json')
+  .expectBadge({
+    label: 'total tests',
+    message: isMetricAllowZero,
+  })
+
+t.create('Passed')
+  .get('/passed/swellaby/swellaby:testspace-sample/main.json')
+  .expectBadge({
+    label: 'passed tests',
+    message: isMetricAllowZero,
+  })
+
+t.create('Failed')
+  .get('/failed/swellaby/swellaby:testspace-sample/main.json')
+  .expectBadge({
+    label: 'failed tests',
+    message: isMetricAllowZero,
+  })
+
+t.create('Skipped')
+  .get('/skipped/swellaby/swellaby:testspace-sample/main.json')
+  .expectBadge({
+    label: 'skipped tests',
+    message: isMetricAllowZero,
+  })
+
+t.create('Errored')
+  .get('/errored/swellaby/swellaby:testspace-sample/main.json')
+  .expectBadge({
+    label: 'errored tests',
+    message: isMetricAllowZero,
+  })
```

Comment:
Are we omitting tests for the not found case(s) here because it is handled in the base class and covered by another test file, or is it oversight?

Reply:
Those tests were added to https://github.com/badges/shields/blob/593a6ace605613a95219804fbc06f397ca378cb2/services/testspace/testspace-test-pass-ratio.tester.js 

Some other maintainers in the past have expressed concerns with having multiple not found cases in instances like this where all the service classes are hitting the same upstream endpoint via a base class, so opted to just stick it in another tester that had fewer scenarios